util/tooltool/tooltool.py

```diff
@@ -392,10 +389,9 @@ def validate_manifest(manifest_file):
         else:
             if not f.validate():
                 invalid_files.append(f)
-    if len(invalid_files + absent_files) == 0:
+    if not invalid_files + absent_files:
```

Comment:
The specific pylint complaint is `Unnecessary parens after u'not' keyword (superfluous-parens)`